[core]/es_extended/common/modules/table.lua

```diff
@@ -133,6 +133,25 @@ function ESX.Table.Join(t, sep)
 	return str
 end
 
+-- Credits: https://github.com/JonasDev99/qb-garages/blob/b0335d67cb72a6b9ac60f62a87fb3946f5c2f33d/server/main.lua#L5
+function ESX.Table.TableContains(tab, val)
+    if type(val) == "table" then
+        for _, value in ipairs(tab) do
```

Comment:
Be careful with the ipairs operation, it does not work in all cases https://stackoverflow.com/questions/55108794/what-is-the-difference-between-pairs-and-ipairs-in-lua